plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestAtopPlugin.java

```diff
@@ -27,5 +34,14 @@ public void testGetConnectorFactory()
         assertInstanceOf(getOnlyElement(plugin.getConnectorFactories()), AtopConnectorFactory.class);
     }
 
-    // TODO test factory
+    @Test
+    public void testCreateConnector()
```

Comment:
We can rename testGetConnectorFactory to `testCreateConnector` and reuse it instead.

Reply:
renamed